transcom/mymove
# MAIN B-20966 - tool-tips-missing-from-SIT-service-items

## [B-20966](https://www13.v1host.com/USTRANSCOM38/assetdetail.v1?number=B-20966)

INT PR: https://github.com/transcom/mymove/pull/13811

## Summary

This is to add missing tooltips to SIT related service items in 'Move task order' page. These tooltips only show when a rejected SIT related service item is requested again using the 'Edit' page in Prime Sim. There were three service items missing the tooltips - `Domestic Destination Add'l SIT`, `Domestic Destination SIT Fuel Surcharge`, and `Domestic Origin SIT Fuel Surcharge`.

Also redesigned the tooltip so the user doesn't have to scroll and also to somewhat match the mockup given in the attachment in agility.


### How to test

1. Use `HHGMoveInSITEndsTomorrow` to create a move with SIT service items added already.
2. As a TOO -> search for the move and go to 'Move task order' tab.
3. Scroll down and find Approved Service Items list and click Reject and submit for all 8 SIT related items.
4. In Prime Sim -> search for the move and scroll down to get list of Service Items.
5. You should see 'Edit' button under each SIT related service items.
6. Click 'Edit' and fill in only the 'Update Reason' and click 'Save'.
7. Go back to TOO -> go to 'Move task order' page for the move -> Scroll down to 'Requested Service Items' section and you should see tooltips beside each SIT related items.
8. Hover over the tooltip icon and you should see the header and content similar to one is the screenshot.

## Screenshots
## BEFORE
![image](https://github.com/user-attachments/assets/8d3fd131-5d1b-4a77-b582-6263f97b0eb5)
![image](https://github.com/user-attachments/assets/7c48f13d-9994-46e4-968e-342a26084460)

## AFTER
![image](https://github.com/user-attachments/assets/bb742a0b-ba31-47df-ac39-3c33a9825e5b)
![image](https://github.com/user-attachments/assets/ca5e6bee-c0e0-4a5f-bf86-bd444065eb7e)
![image](https://github.com/user-attachments/assets/f8af1483-1e36-4b30-9a8e-a8c1c4dc7a00)
